.github/workflows/go-test.yaml

```
@@ -73,7 +73,15 @@ jobs:
       - name: Rbd command
         run: |
           set -ex
-          kubectl rook-ceph rbd ls replicapool
+          SUCCEEDED=0
+          for ((i=0;i<30;i++)) ; do
+            timeout 10 kubectl rook-ceph rbd ls replicapool || continue
+            SUCCEEDED=1
+          done
+          if [ "${SUCCEEDED}" -ne 1 ]; then
+            echo "running rbd command timed out"
+            exit 1
+          fi
```

```suggestion
          SUCCEEDED=0
          for ((i=0;i<30;i++)) ; do
            timeout 10 kubectl rook-ceph rbd ls replicapool || continue
            SUCCEEDED=1
          done
          if [ "${SUCCEEDED}" -ne 1 ]; then
            echo "running rbd command timed out"
            exit 1
          fi
```


I think better option is check move the command before mon restore and check if `replicapool` status is ready other wise command will stuck and this is what happening
```bash
 replicapool   Failure
```

wdyt @satoru-takeuchi ?

---

Do you mean that the better way is just exchanging the order of "Mon restore" section and "Rbd command" section?

---

yes